src/core/libmaven/mzSample.h

```diff
@@ -217,6 +217,16 @@ class mzSample
     * @brief Load sample (supported formats: .mzxml, .mzml, .mzdata, .mzcsv and .cdf)
     * @param filename Sample file name
     */
+
+    enum class SampleType: int {
+        NONE = 0,
+        MS,
+        SRM,
```

Comment:
There's no need for separate `SRM` and `MRM` types imo. Just calling it MRM is fine.